README.md

```diff
@@ -26,24 +26,24 @@ You can add this charts repo by:
 helm repo add skyscrapers https://skyscrapers.github.io/charts
 ```
 
-The actual Helm charts index is being hosted in GitHub pages in this same repo (`gh-pages` git branch). 
+The actual Helm charts index is being hosted in GitHub pages in this same repo (`gh-pages` git branch).
 The charts and the charts index are automatically updated by Concourse when new commits arrive
 on the master branch. see the [`ci`](https://github.com/skyscrapers/ci) repository for more details
 on the setup.
 
 ## Bootstrap base charts
 
-First you need to generate a `values.yaml` using the 
+First you need to generate a `values.yaml` using the
 [`terraform-kubernetes/base`](https://github.com/skyscrapers/terraform-kubernetes/tree/master/base)
 module, then you can just start installing:
 
 ```console
-helm upgrade --install kube2iam skyscrapers/kube2iam --values helm-values.yaml
-helm upgrade --install kube-lego skyscrapers/kube-lego --values helm-values.yaml
-helm upgrade --install nginx-ingress skyscrapers/nginx-ingress --values helm-values.yaml
-helm upgrade --install external-dns skyscrapers/external-dns --values helm-values.yaml
-helm upgrade --install kubesignin skyscrapers/kubesignin --values helm-values.yaml
+helm upgrade --install kube2iam --namespace infrastructure --values helm-values.yaml
+helm upgrade --install kube-lego --namespace infrastructure --values helm-values.yaml
+helm upgrade --install nginx-ingress --namespace infrastructure --values helm-values.yaml
```

Comment:
it should work. from the nginx-ingress docs: `--watch-namespace string           Namespace to watch for Ingress. Default is to watch all namespaces`